pygmt/figure.py

```diff
@@ -7,7 +7,7 @@
 
 try:
     import IPython
-except KeyError:
+except ModuleNotFoundError:
     IPython = None  # pylint: disable=invalid-name
```

Comment:
I can't see any reporting on Codecov (because the codecov upload doesn't work with test failures due to #1217), but I'm assuming this line has code coverage?